### PR TITLE
✨ order series

### PR DESCRIPTION
**Pull request by roiLeo**
Will it blend?

### CODE REVIEW

File: src/resolvers/series.ts [hunk 1]
```diff
@@ -32,9 +36,11 @@ export class SeriesResolver {
       JOIN event e on ne.id = e.nft_id
       WHERE e.interaction = 'BUY'
       GROUP BY ce.id, me.image, ce.name 
-      ORDER BY total DESC
-      LIMIT $1 OFFSET $2;
-    `, [limit, offset])
+      ORDER BY $1 $2
```
Comment (vikiival) [inline src/resolvers/series.ts:39]: Can we maybe leverage interfaces for this ? 

```ts
enum OrderBy {
  DESC = 'DESC',
  ASC = 'ASC'
}
```

with string we are bit prone to some sql hack

**Review by vikiival — CHANGES_REQUESTED** [review]
Can we blend?

File: src/resolvers/series.ts (hunk 1 above)
Comment (roiLeo) [reply src/resolvers/series.ts:55]: 🏴‍☠️ Never trust user input

**Comment by vikiival** [comment]
Screenshot 2022-01-06 at 12 22 57

**Comment by roiLeo** [comment]
@vikiival that wasn't working because the query was transform like:
```
ORDER BY 'total' 'ASC'
```
instead of:
```
ORDER BY total ASC
```
we have to build query dinamicly beforehand.

I've managed to setup the project, will do same change on spotlight next.
Screenshot 2022-01-06 at 17-08-01 Query node playground

**Comment by yangwao** [comment]
viki has said that maybe we need to turn of server sorting something he is not opening mouth really loud so translating literally through my brain to keyboard at evening and it should work for series insights 👯‍♂️

**Comment by roiLeo** [comment]
> @roiLeo I'm still getting weird behaviour 

because
> LIMIT undefined OFFSET undefined

we could try to setup default value or update `nullable` to `false`

**Comment by vikiival** [comment]
@roiLeo fixed 👁️

**Review by vikiival — APPROVED** [review]

**Comment by roiLeo** [comment]
Something will not work here, `unique` is reserved sql term, it won't be able to order this column

**Comment by roiLeo** [comment]
Capture d’écran 2022-01-17 à 3 35 42 PM

**Pull request merged**